xbmc/music/tags/TagLoaderTagLib.cpp

```diff
@@ -1048,10 +1057,19 @@ void CTagLoaderTagLib::SetGenre(CMusicInfoTag &tag, const std::vector<std::strin
 
 void CTagLoaderTagLib::SetReleaseType(CMusicInfoTag &tag, const std::vector<std::string> &values)
 {
+  tag.SetBoxset(false);
   if (values.size() == 1)
+  {
+    if (values[0].find("boxset") != std::string::npos)
```

Comment:
Think "boxset" extraction from album type tag should not happen in a tag factory call

Reply:
OK, moved to CMusicInfoScanner::CheckBoxSets